model/BUILD

```diff
@@ -4,6 +4,7 @@ go_library(
     name = "go_default_library",
     srcs = [
         "controller.go",
+        "ingress.go",
         "registry.go",
```

Comment:
Please remove ingress from the model. Model is platform agnostic representation of resources. Mesos may not even have ingress.